README.md

```diff
@@ -1,5 +1,18 @@
 ### Arrays
 
+#### First
+
+* Underscore
+  ```javascript
+  _.first([5, 4, 3, 2, 1]);
+  ```
+
+* ES2015
+
+  ```javascript
+  const [n] = [5, 4, 3, 2, 1];
+  ```
+
```

Comment:
Yeah, I agree. Do you think it's worth including `const first = ([n]) => n;` as a simple definition of `first`?